swagger.yaml

```diff
@@ -0,0 +1,244 @@
+swagger: "2.0"
+info:
+  description: "discretionary-business-grants API Schema"
+  version: "0.0.1"
+  title: "discretionary-business-grants API"
+host: "discretionarybusinessgrants.hackney.gov.uk"
+basePath: "/api"
+tags:
+  - name: "urls"
+    description: "Secure URL for uploading files"
+  - name: "applications"
+    description: ""
+schemes:
+  - "https"
+paths:
+  /urls:
+    post:
+      tags:
+        - "urls"
+      summary: "Retrieve a new secure URL for file upload"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: "body"
+          in: "body"
+          required: true
+          schema:
+            $ref: "#/definitions/Url"
+      responses:
+        "500":
+          description: "Invalid input"
+        "200":
+          description: "Success"
+          schema:
+            type: "object"
+            items:
+              $ref: "#/definitions/UrlResponse"
+  /applications:
+    post:
+      tags:
+        - "applications"
+      summary: "Create a new business grant application"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: "body"
+          in: "body"
+          required: true
+          schema:
+            $ref: "#/definitions/Applications"
+      responses:
+        "500":
```

Comment:
As above. We probably want to keep 500 for server error and move invalid input to 400